lib/OptimizationMOI/src/nlp.jl

```diff
@@ -85,7 +85,7 @@ function SciMLBase.get_syms(sol::SciMLBase.OptimizationSolution{
     C <:
     MOIOptimizationNLPCache,
 }
-    sol.cache.evaluator.f.syms
+    variable_symbols(sol.cache.evaluator.f)
```

Comment:
Doesn't OptimizationMOI need its Project.toml updated?

Reply:
Bump on this.

Reply:
Ah, I didn't realize. Thanks. I'll push these changes to the `rerun CI` PR